### PR TITLE
Trimming quotes off browser user agent string

### PR DESCRIPTION
**Pull request by burzum619**
This user agent string is used by the TRT, and is expected to be unquoted

TDS-1034

### CODE REVIEW

File: service/src/main/java/tds/exam/services/impl/ExamServiceImpl.java [hunk 1]
```diff
@@ -451,8 +451,9 @@ public void pauseAllExamsInSession(final UUID sessionId) {
         /* StudentDLL [5344] Skipping getInitialAbility() call here - the ability is retrieved in legacy but never set on TestConfig */
 
         if (exam.getStartedAt() == null) { // Start a new exam
-            // Initialize the segments in the exam and get the testlength.
-            Exam initializedExam = initializeExam(exam, assessment, browserUserAgent);
+            // Initialize the segments in the exam and get the testlength and trim the quotes off the user agent string
+            String unquotedUserAgent = browserUserAgent.replaceAll("^\"|\"$", "");
```
Comment (jtreuting) [inline service/src/main/java/tds/exam/services/impl/ExamServiceImpl.java]: I guess I'm wondering why it is quoted in the first place?  Is it more appropriate to fix it at the source (depending on what that source is).

Comment (burzum619) [inline service/src/main/java/tds/exam/services/impl/ExamServiceImpl.java]: This string is stored (in /openTest) and then retrieved from the student cookie on the TDS_Student side of things - I'm guessing that when its fetched back from the cookie, it comes back quoted. Do you think it makes more sense to fix this on the Student side instead?

Comment (millerts) [inline service/src/main/java/tds/exam/services/impl/ExamServiceImpl.java]: My vote is to fix it in student as well.  I feel we can leave this here as a precaution but student shouldn't be sending the browser string as quoted

**Comment by burzum619** [comment]
Closing this out to fix on student side

**Comment by millerts** [comment]
Merge it.

**Pull request merged**